context/src/main/java/io/grpc/Context.java

```diff
@@ -185,8 +185,8 @@ public static Context current() {
     return current;
   }
 
-  private ArrayList<ExecutableListener> listeners;
-  private CancellationListener parentListener = new ParentListener();
+  ArrayList<ExecutableListener> listeners;
+  CancellationListener parentListener = new ParentListener();
```

Comment:
The fields of private inner classes are fine, since this PR doesn't make them visible to outside of the file. `checkNotNull()` and `lookup()` and also fine, since no harm can be done with it. I am a little uneasy about making these two mutable fields package-private, although we are unlikely to mess with it outside of the class. @ejona86 WDYT?

Reply:
Why are these being changed? `listeners` and `parentListener` is only referenced directly by Context.

Reply:
Accidentally marked public while trying to reverse the field from javap.  PTAL